### DISCUSSION TITLE
Articles » Category Blog with tag changes all articles' url path

### DISCUSSION THREAD

**Discussion by shawnhy** [joomla/joomla-cms#47058] [2. Cold Features]
### Steps to reproduce the issue
1. Create a category named: News.
2. Create an article in category News and named: "EOL notification".
3. Add a tag "eol" to above article.
4. Create second article in category News and named: "Normal news". We will not apply any tag to this article
5. Create Articles » Category Blog menu item, named: "EOL notifications(alias: eol-notifications)". We select category "News" and tag "eol" as the contidtions to display articles.
6. Create another Articles » Category Blog menu item named: News(alisa news). Yes we select category "News" so that two articles will display here.
7. From frontend you will see the URLs of both article will be like /eol-notifications/articlealias...This causes confusing information because the second article is not EOL-related.

### Expected result
Articles should keep their path and not be affected by tag conditions in menu item

### Actual result
the URLs of both article will be like /eol-notifications/articlealias

### System information (as much as possible)
Joomla 4.04
php 8.2

**Comment by ghost** [comment]
Got the same result on 4.4.2. I guess the result is because you have 2 menu items using the same category.

**Comment by Hackwar** [comment]
This is actually a feature request, since you are expecting the router to take the tags into consideration for routing. This would be a new feature.
